core/src/main/java/io/lionweb/lioncore/java/language/PrimitiveType.java

```diff
@@ -37,7 +37,7 @@ public PrimitiveType(@Nullable Language language, @Nullable String name, String
 
   @Override
   public String toString() {
-    return "PrimitiveType(" + getName() + ")";
+    return super.toString() + "{" + "name=" + getName() + '}';
```

Comment:
This is the only `LanguageEntity` that has a special `toString()`. Why?

Reply:
Right, let's remove it